Libs/LibUIDropDownMenu/LibUIDropDownMenu.lua

```diff
@@ -27,80 +27,82 @@ local lib = LibStub:NewLibrary(MAJOR_VERSION, MINOR_VERSION)
 if not lib then return end
 
 -- Determine WoW TOC Version
-local WoWClassic, WoWRetail
+local WoWClassicEra, WoWClassicTBC, WoWRetail
 local wowtocversion  = select(4, GetBuildInfo())
-if wowtocversion < 30000 then
-	WoWClassic = true
+if wowtocversion < 20000 then
+	WoWClassicEra = true
+elseif wowtocversion > 19999 and wowtocversion < 90000 then 
+	WoWClassicTBC = true
 else
 	WoWRetail = true
 end
 
-if WoWClassic then
+if WoWClassicEra or WoWClassicTBC then
 	GameTooltip = _G.GameTooltip
 	tooltip = GameTooltip
-else -- Shadowlands
+else -- Retail
 	GetAppropriateTooltip = _G.GetAppropriateTooltip
 	tooltip = GetAppropriateTooltip()
 	GetValueOrCallFunction = _G.GetValueOrCallFunction
 end
 
 -- //////////////////////////////////////////////////////////////
-L_UIDROPDOWNMENUQUESTIE_MAXBUTTONS = 1;
-L_UIDROPDOWNMENUQUESTIE_MAXLEVELS = 2;
-L_UIDROPDOWNMENUQUESTIE_BUTTON_HEIGHT = 16;
-L_UIDROPDOWNMENUQUESTIE_BORDER_HEIGHT = 15;
+L_UIDROPDOWNMENU_MAXBUTTONS = 1;
+L_UIDROPDOWNMENU_MAXLEVELS = 2;
+L_UIDROPDOWNMENU_BUTTON_HEIGHT = 16;
+L_UIDROPDOWNMENU_BORDER_HEIGHT = 15;
```

Comment:
Renaming got lost purposely as the old refork and #3150 refork creates new bugs compared to the original library because refork is not as easy as just renaming variables. I didn't feel to review whole library for all necessary additional changes after finding first ones. In theory reviewing all lines near changed lines would be enough.

Reply:
This was first and then I decided better to use working one and see if the lib solves all taint issues, which it didn't solve. So I left original and went to solve rest instead using time to try review the lib.
https://github.com/Questie/Questie/blob/639366151bb48b0af09aed68fdfb47a38f240ced/Libs/LibUIDropDownMenu/LibUIDropDownMenu.lua#L1507

Reply:
Fork done again in https://github.com/Questie/Questie/pull/3154/commits/3b7d8b3c744ed71d16d939cd556fd4988393debb